client/post-editor/editor-more-options/copy-post.jsx

```diff
@@ -66,6 +66,11 @@ class EditorMoreOptionsCopyPost extends Component {
 
 	render() {
 		const { translate, siteId } = this.props;
+
+		if ( ! siteId ) {
```

Comment:
Per the reactivity guidelines, it's best if you implement components without consideration of any particular linear user flow, but rather to be able to render reasonably well with whatever happens to be the state of the application at any particular time. The flows mindset tends to lead to these sorts of issues and makes the components far less portable.